js/template-card.js

```diff
@@ -0,0 +1,97 @@
+import {createOffers} from './util.js';
+
+const templateFragment = document.querySelector('#card').content;
+const template = templateFragment.querySelector('.popup');
+const fragment = document.createDocumentFragment();
+const similarOffers  = createOffers();
+
+similarOffers.forEach(({author, offer}) => {
+  const advertElement = template.cloneNode(true);
+  const offerTypeAssociatedWithSignatures = {
```

Comment:
Это сейчас создается на каждый вызов в `forEach`
В этом смысла нету. Просто вынеси его выше
```suggestion
  const offerTypeToReadable = {
```